.travis.yml

```diff
@@ -1,33 +1,26 @@
+        ExgApV1VEnJ4ne0NHi/SEMPnuCOHBBsbXY6gGTcBoAKJ3ms9w0MDzH4qRroKtoGjC4xO7JFnvi+FE/3iF1rpG3y+6kjl7oTlB8lcJZsUUrLG10H8Q1mGmLS18D25PF93qyi/xDmPci+KarHoUID61sTJu/8nqgmBdzTBqOQaaJAcjBM73t6k/9zYr2U8dx5X1Lalddg2u1F4p6IUy518bjDr1ReWq/JSD54NODYuRZWKOLHfIaeHQQCS4u+wjv/7WyFP0crlqrVypvrKZn3ZrBIEKRM8X5cXLNv/fIluvw0YboItv1Uwo+pWwz2pFIfy80FvN0slm1CcbRTRTLc8Nfx4CBuLojz2ElnVseNZKP2Z1EgsT6l/EuHHlINCwOhWHElVM5NVTjIcUPLGn+lnw1slotB7HIjHBgzIr9217lIyLAoswL6e8ztb0o+4yy5NwzzaOuwh9M5ndKLQJEfHEzBbQUDM/oF3kbmSWAgJHSeJhthoyJ16rwlXUOfH7Vlscc8LuYu5qHDrLD9vulPCqCZQpfECjum2bKs4bk4Cxf0//iWBAm0T8XXgviownDYvTihPYpEvEtEdeFGNu9vxuTAOr0KbdkJwMHVrSKUSq3wC+zU9bjBCXFDubF+KHtoLHo/AkEIAkZ0c7kEDBxbC7OG5+lNFlPoYXz2qA9S45Mk=
     - secure: >-
-        Q3xN3fRvTQuy/HKlkYFFnVrFo01r6Q8zgzHHK8yKNKio9T/BM0+iIMYP8mRY+qvCQxCboXuUawG6gsxxT2Zn/p7SgNZ+UHq7DcLmocqxmECdGfqra6Hy9Y7BZVYPWlWIADkNUjI0RfOz69/3o/TAFlt4Cnw3BX3ip0o0Rri5/jzj0Nn+xSF+onyMnpH2gQvUE77MHvCuWTPki3R86Bz8RRzbTwKYwVa0oVca7jzdPtOjqzsnz8k9X0HVwQEpHGjqgTP3lg8EotON0rac/ayWs3J3bk9ye5AfvdTCYcZnDcz8POAN6FeC2Xey7oqQO4N8vFagru88mC3AmN67ZYqwI0fmDEYre20lnXFAFBFzsiu4FvKMgFYi/C5tG5ngSR5XcXwLskax82Un7yYYRzVbyhFSxxWZCqIC4cJ5d+E0Wex4eEYDwAlZ9AFP/hRJ2qrBVUbzXeGVMkHlg4qtdUIzHRkMUmjvEWVzTvk0Xa2wD+S0mYPcBwQcscoZy9zqdJM4hfbG9IGi02/DyrTOL3IuNH8WHHvGWS4ajDhSRkGCV4I5VtGKQ/Y3RA/pU6AaI7tVcDhbxOyIPKuOXO6PWuZOiJhhPQ/kceMkUXeh0TmsDtSBpe5OKCnjsdSn02lTT5yYf4nlVIVczP9p6zBerXorzZk1/S3CRBtqYZPgRrefbCw=
-node_js: stable
+        kYbI9qDVdminzjmND0waDuLvHtJOJl2Po/XJTpLZ9H01d+jkFHzi6cTLYTh6TOwEdictadQ3mJdOEO4JTufCGS9Z7mHPFegF7vnWqefn/s+7eZMqQRggydqyUL+XeH6AgUOdjawORKR4BweT/TOavnyJ8trtEEnafiHVfw35ltT81QLFkob20zzB8lsh9asTC3ktC+m3hes2vEj9RuIusGd9kYykbqXhm3Wzpg1n82VtRYghkSxBC0KmGCYwZJLDSkd8ExjI/dzfYlYyV8L9/lZhav9PRDLysD5wo4YQKBJ7GQ+yKXuItaeKz2aaVpBqUhwlqs+AXoebpww6DNAUw6ZBAQYJodVBlVrDYANFaRZgLcWT4aIYpbd65aZqHM7tN8gA86j4uYT57Q1PZrsyl86yK53GFreLOpe5MtWaf3jxXzmx7q+M5DeVy9maXMXIQLx4KLJs2gOKog43CZ0Ww290wRpuRTC6GZTMYKTaYedHkFVcxOJwODeR0Uapgt+nCticJ/AI+vjWS4iaN3G7YHgRUBoOkNYCKX1JrjMyQ/bF3A3UVyNkX4j4GP42QfxAx0J3M3xjmExVKA1zVPLzHkLLB3ssE6VIlZkpfTmvTUZu5v1x9b576HB4vXlfBdqMAOFFPc0Soyng8RC6V1W6qxARxP+tpQPYBAl0kqACqVA=
+node_js: '9'
 addons:
   firefox: latest
-  apt:
-    sources:
-      - google-chrome
-    packages:
-      - google-chrome-stable
-  sauce_connect: true
+  chrome: stable
 script:
   - xvfb-run polymer test
-  - >-
+  - |-
+    >-
```

Comment:
The `>-` is now passed to the shell:
https://travis-ci.org/PolymerElements/iron-checked-element-behavior/builds/382468360#L1108
I don't know enough about YAML to recommend a fix though. :/ What was this change meant to do?

Reply:
I don't know, I just ran what was default on the travis pass on tedium. This may be a question for @rictic or @azakus

Reply:
Here's a fun site I just found: https://yaml-multiline.info/
It looks like `|-` is "keep all the newlines, except for trailing ones" and the `>-` is "remove all the newlines". I think this means that `>-` can be removed. I'm going to try pushing some changes on top of this branch to see what happens.